acceso/urls.py

```diff
@@ -17,6 +16,5 @@
 
     # jsGrid
     path('caso-index/', views.caso_index, name='caso_index'),
-    path('api/', casoTable.as_view()),
-    re_path(r'api/(?P<caso_id>[0-9])/$', casoTable.as_view()),
+    path('api/<int:caso_id>/', views.caso_table),
```

Comment:
Use `path` instead of `re_path` when you can. The syntax is simpler, easier to read, and harder to get wrong.